integration_tests/test_str_attributes.py

```diff
@@ -182,6 +182,108 @@ def partition():
     seperator = "apple"
     assert s.partition(seperator) == ("rendezvous 5", "", "")
 
+def islower():
```

Comment:
I think the error might be caused by this?
Try using different name for the function.

Reply:
You can use `lpython a.py --show-c` to see the C code.

Reply:
Yep. That fixed it. Thanks